src/style.css

```diff
@@ -1 +1,90 @@
-/* add styles here */
+body {
+  font-family: ' Avenir ', sans-serif;
+  font-weight: 300;
+}
+
+input {
+  all: unset;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+}
+
+@font-face {
+  font-family: ' Avenir ';
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+}
```

Comment:
The `@font-face` rule should also include a font-weight property to specify the weight of the font being defined. This helps in properly selecting the right font when different weights are used in the CSS.